datasets/at/pep/crawler.py

```diff
@@ -0,0 +1,68 @@
+from dateutil.parser import parse
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+
+def parse_german_date(date_string):
+    return parse(date_string, dayfirst=True, locale='de')
+
+def crawl_item(url_info_page: str, context: Context):
+
+    info_page = context.fetch_html(url_info_page)
+
+    first_name = info_page.xpath("//span[@itemprop='http://schema.org/givenName']/text()")[0]
+    last_name = info_page.xpath("//span[@itemprop='http://schema.org/familyName']/text()")[0]
+
+    person = context.make("Person")
+    person.id = context.make_id(first_name, last_name)
+
+    person.add("firstName", first_name)
+    person.add("lastName", last_name)
+
+    person.add("website", url_info_page)
```

Comment:
I think this fits better in the `sourceUrl` property, while `website` is more suited to their own website, if they had one.